### PR TITLE
Create a deposit progress component

### PR DESCRIPTION
**Pull request by jcoyne**
This helps illustrate that motion is working to solve the kind of problems we want it to.

Screen Shot 2020-09-14 at 9 08 39 PM

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -19,4 +19,7 @@ import 'stylesheets/main'
 //
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
-import "motion"
\ No newline at end of file
+import "motion"
+
```
Comment (mjgiarlo) [inline app/javascript/packs/application.js:23]: double blank line but nbd

File: app/javascript/modules/validate-forms.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Example starter JavaScript for disabling form submissions if there are invalid fields
+(function () {
+  'use strict'
+  window.addEventListener('turbolinks:load', function() {
+    // Fetch all the forms we want to apply custom Bootstrap validation styles to
+    var forms = document.querySelectorAll('.needs-validation')
+
+    // Loop over them and prevent submission
```
Comment (mjgiarlo) [inline app/javascript/modules/validate-forms.js:8]: can you say more about using JavaScript for client-side validation over Motion?

Comment (jcoyne) [reply app/javascript/modules/validate-forms.js:8]: By default browsers do client side validation.  This removes the ugly browser default and adds the bootstrap way. We can also do validation with Motion.

File: app/javascript/packs/application.js (hunk 1 above)
Comment (jcoyne) [reply app/javascript/packs/application.js:23]: ESLint is that you?

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**